processing/app/core/exceptions.py

```diff
@@ -0,0 +1,32 @@
+from typing import Any
+from typing import Dict
+from typing import Optional
+
+from fastapi import HTTPException
+from fastapi import status
+
+
+class DuplicatedError(HTTPException):
+    def __init__(self, detail: Any = None, headers: Optional[Dict[str, Any]] = None) -> None:
+        super().__init__(status.HTTP_400_BAD_REQUEST, detail, headers)
+
+
+class AuthError(HTTPException):
+    def __init__(self, detail: Any = None, headers: Optional[Dict[str, Any]] = None) -> None:
+        super().__init__(status.HTTP_403_FORBIDDEN, detail, headers)
+
+
+class NotFoundError(HTTPException):
+    def __init__(self, detail: Any = None, headers: Optional[Dict[str, Any]] = None) -> None:
+        super().__init__(status.HTTP_404_NOT_FOUND, detail, headers)
+
+
+class ValidationError(HTTPException):
+    def __init__(self, detail: Any = None, headers: Optional[Dict[str, Any]] = None) -> None:
+        super().__init__(status.HTTP_422_UNPROCESSABLE_ENTITY, detail, headers)
+
+
+# if get file for task_id that has no result
```

Comment:
Remove the comment. It doesn't really matter here why the error is thrown, and it's probably soon that the comment will be outdated and someone would have to remember that it was here.